ts/repositories.bzl

```diff
@@ -9,34 +9,18 @@ load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 load("//ts/private:npm_repositories.bzl", "npm_dependencies")
 load("//ts/private:versions.bzl", "TOOL_VERSIONS")
 
+LATEST_TYPESCRIPT_VERSION = TOOL_VERSIONS.keys()[-1]
+
 def http_archive(**kwargs):
     maybe(_http_archive, **kwargs)
 
-LATEST_TYPESCRIPT_VERSION = TOOL_VERSIONS.keys()[-1]
-
 # WARNING: any additions to this function may be BREAKING CHANGES for users
 # because we'll fetch a dependency which may be different from one that
 # they were previously fetching later in their WORKSPACE setup, and now
 # ours took precedence. Such breakages are challenging for users, so any
 # changes in this function should be marked as BREAKING in the commit message
 # and released only in semver majors.
-def rules_ts_dependencies(ts_version_from = None, ts_version = None, ts_integrity = None):
-    """Dependencies needed by users of rules_ts.
-
-    To skip fetching the typescript package, define repository called `npm_typescript` before calling this.
-
-    Args:
-        ts_version_from: label of a json file (typically `package.json`) which declares an exact typescript version
-            in a dependencies or devDependencies property.
-            Exactly one of `ts_version` or `ts_version_from` must be set.
-        ts_version: version of the TypeScript compiler.
-            Exactly one of `ts_version` or `ts_version_from` must be set.
-        ts_integrity: integrity hash for the npm package.
-            By default, uses values mirrored into rules_ts.
-            For example, to get the integrity of version 4.6.3 you could run
-            `curl --silent https://registry.npmjs.org/typescript/4.6.3 | jq -r '.dist.integrity'`
-    """
-
+def rules_ts_bazel_dependencies():
```

Comment:
Can we put a `_` to make it clear this one is internal?

Reply:
No, because then we'd get an error calling it from our WORKSPACE